casper/src/test/resources/RevAddressTest.rho

```diff
@@ -29,10 +31,41 @@ in {
   } |
 
   contract test_valid_address(rhoSpec, _, ackCh) = {
-    stdlog!("info", "test valid address") |
     new retCh in {
       revAddress!("validate", "1111K9MczqzZrNkUNmNGrNFyz7F7LiCUgaCHXd28g2k5PxiaNuCAi", *retCh) |
       rhoSpec!("assert", (Nil, "== <-", *retCh), "expecting a Nil value since there are no errors", *ackCh)
     }
+  } |
+
+  contract test_fromPublicKey(rhoSpec, _, ackCh) = {
+    new r1, r2, r3 in {
+      revAddress!(
+        "fromPublicKey",
+        "0000000000000000000000000000000000000000000000000000000000000000".hexToBytes(),
+        *r1
+      ) |
+      revAddress!(
+        "fromPublicKey",
+        "1111111111111111111111111111111111111111111111111111111111111111".hexToBytes(),
+        *r2
+      ) |
+      revAddress!(
+        "fromPublicKey",
+        "2222222222222222222222222222222222222222222222222222222222222222".hexToBytes(),
+        *r3
+      ) |
+      rhoSpec!("assertMany", [
+        (("111123jwD7acNy6vfYwzWXAbnSQ8z8Bv24cCkVcVbd6SuP2LtFK7w7", "== <-", *r1), "correct RevAddress"),
+        (("11112comWwT626KfXkvnWRHDwPDm4VefqSr6ijvinhVHGTuXe6wWmk", "== <-", *r2), "correct RevAddress"),
+        (("11112ipm5NY9BqyQ3p3sUx8pdFWV5jFQ5qqMcUzZWe19hwz3USbHBG", "== <-", *r3), "correct RevAddress"),
```

Comment:
In this case three tests don't bring more value than one test

Reply:
Having all those has the advantage of making the magic strings used in the header of RevVaultTest.rho searchable in the codebase, which saves work to anybody willing to validate if the strings used there are indeed valid.